course/meetings/eeb-asher-martin-2018-02-14.md

```diff
@@ -0,0 +1,122 @@
+-   Have a formal presentation for the final project
+    -   Angle as an end of semester symposium for the students (since no one took up
+        the offer to submit officially this year, we will make a presentation a
+        lecture component for last year).
+-   More updates during the project work, have a set of questions that the student
+    answer at the beginning of each class to make sure they share their struggles
+    and surceases with the other groups
+-   We are free to reuse any assignments we want, just take them down from the
+    homepage.
+
+
+<a id="org9e6dffd"></a>
+
+## Teaching setup
+
+-   There will probably be around 3-4 of us and 0-1 grad students from EEB.
+-   We could either be TAs as last year or have a setup where 1-2 of us are course
+    (co)-instructors and then 1-2 TAs.
+    -   Course instructor positions have to be posted, but are usually written so that
+        it is difficult for not difficultly.
+    -   However, Asher prefers someone with past experience to be the course
+        instructor, so not Ahmed as we originally though =(
+        -   Although, I think if we have two co-instructors, maybe it could be
+            Ahmed together with someone from last fall.
+    -   In addition to this we could always include guest lecturers from UofTCoders
+        if we need more help.
+    -   It is up to us how the pay will be divided, but usually TA help is not given
+        to course under 25 students so can't demand too much.
+
+
+<a id="org90ef1d8"></a>
+
+## Content redesign
+
+-   Important for next year is to make sure that the analysis techniques required
+    for the group work are all covered during the class ours.
+-   In the student feedback, many wished they had additional lecture time focused
+    on mixed effects modelling.
+-   Asher thinks it's ok to keep the diversity of data sets (and not make it only
+    time series for example).
+    -   They get more ownership over research question, but it does depend on the
+        objective of the course.
+-   A good idea could be to select 2-3 types data sets (where "type" refers to
+    that they would require distinct analysis methods) with 1-2 data sets each.
+    -   Make sure that each of these analyses techniques are covered during class.
+-   To accommodate this deeper focus on the modelling techniques, the part that
+    Madeleine taught last year will be modified.
+    -   Many student appreciated learning the modelling component from the very
+        basics, but also expressed some confusion for the relevance of this in
+        regards to the analysis they performed during the group work.
+        -   More focus on generalized linear models, mixed effect models, 
+            time series, simulations, permutations, non-normal data, etc could be good
+            to include here.
+-   Probably we need more time for these new parts, so we might exclude the
+    Git/GitHub lectures as well.
```

Comment:
hm, interesting. How will we get them to collaborate then? Through Google Drive or something? I'd really not like that..

Reply:
Yes, something like Dropbox, or Drive. No doubt that this is not as good as Git for collaborating on code and dealing with conflicting versions etc, but it might be good enough for students that are just learning programming for the first time.

IIRC, Much of our group support time was helping with Git-related tasks and at least three of the students wanted more time to learn it properly (brought it up multiple times in the evaluation).

I think the way they worked, they didn't end up needing something as powerful as Git. Lots of their work during class was multiple people sitting together together and one who coded, and work at home was often on separate files/areas of the project for the individual team members (it seems like from their commits).

Reply:
Were there thoughts about moving Git to the end portion when it's more project work? We had 6 whole lectures for project work, whereas we can make them Git + Project work since that is when they'll be using it the most?

Reply:
We can definitely still include a lecture or two on Git, yeah. I think we just won't require them to use it for the project.